Scripts/CI/README_Metadata_StyleCheck/entry.py

```diff
@@ -0,0 +1,564 @@
+import os
+import argparse
+import subprocess as sp
+from os import listdir
+from os.path import isfile, join
+import json
+
+def main():
+    # Tell the program that we're using global variables
+    msg = 'Entry point of the docker to run json style check scripts.'
+    parser = argparse.ArgumentParser(description=msg)
+    parser.add_argument('-s', '--string', help='A JSON array of file paths.')
+    args = parser.parse_args()
+    cleanstring = ""
+    try:
+        for letter in args.string:
+            if letter not in '[" \']':
+                cleanstring+=letter
+        file_list = cleanstring.split(",")
+    except Exception as e:
+        print(f"Unable to split args {args.string}.\nException: {e}")
+        exit(1)
+    check_files(file_list)
+    
+def check_files(file_list):
+    print("Files to check: ")
```

Comment:
optional - not sure how noisy the output is and if the end user submitting the PR will see this all or not, but depending on all of that, it may be worth adding a "verbose output" command line option. This will help for debugging issues but will prevent unneeded console messages otherwise

Reply:
I removed this because the file check list is outputted in a different section of the action results. Generally only a few files were changed in any given PR, so it's not too noisy regardless.